ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Impl.hs

```diff
@@ -518,8 +522,8 @@ getEpochSlot hasFS hashInfo err curEpochInfo getWhat epochSlot =
           GetHeader -> fmap (headerHash,) $
               -- Get just the header
               withFile hasFS epochFile ReadMode $ \eHnd ->
-                -- TODO we cannot check the checksum in this case, as we're
-                -- not reading the whole block
+                -- We cannot check the checksum in this case, as we're not
+                -- reading the whole block
```

Comment:
Should we make a more prominent reminder of this for ourselves for the future? If there is an error in the header, the first anyone would know would be the remote peer we serve the header to disconnecting from us for giving them a header with an invalid signature. So they would never request the full block and we'd not do the full crc check.

Reply:
Will do. I'm still planning to better document the new index format.

We could store a checksum of the header too, but that requires a new index format and migration.